src/api_server/src/config.rs

```diff
@@ -56,7 +66,15 @@ impl TryFrom<&Path> for Config {
     ///        # Only used in Remote Attestation-Service mode
     ///        "as_addr": "http://127.0.0.1:50004",
     ///        # Only used in Native Attestation-Service mode
-    ///        "as_config_file_path": "/etc/as-config.json"
+    ///        "as_config_file_path": "/etc/as-config.json",
+    ///        # Only used in Amber Attestation-Service mode
+    ///        "amber" : {
+    ///            "base_url": "https://amber.com",
+    ///            "api_key": "tBfd5kKX2x9ahbodKV1...",
+    ///            "certs_file": "/etc/amber/amber-certs.txt",
```

Comment:
Could we please add some information on the format of that file?

Reply:
I will add some comments.